app/components/ProductList/aux.js

```diff
@@ -12,10 +12,114 @@
      } 
   }
 
+    const condDirection = (value == prevState.sortBy) ? !prevState.direction : prevState.direction; 
 
+      <span {`{name}=='id' ? className={styles.arrowdown} : className={styles.arrowdow2}`}>
 
-    const condDirection = (value == prevState.sortBy) ? !prevState.direction : prevState.direction; 
 
+App.propTypes = {
+  children: PropTypes.node.isRequired,
+}
 
 
-      <span {`{name}=='id' ? className={styles.arrowdown} : className={styles.arrowdow2}`}>
+
+car component 
```

Comment:
There is a lot of funny stuff here, like this kind of comment. I'm surprised, that there is no error, but I guess this file is not used...